### DISCUSSION TITLE
Can piezo-electric behavior be modeled?

### DISCUSSION THREAD

**Discussion by oldninja** [peridigm/peridigm#352] [Q&A] [answered]
Hello dear Peridigm community,
I'm looking at a few problems for my research, and one requires the modeling of piezo-electric behavior. I've seen there are "analysisHasMultiphysics" flags in the core peridigm.cpp; however, I need to find out how developed this interphase is and what the process would be to extend it to include piezo-electric behavior. 
@diehlpk @djlittl Do you know anyone familiar enough with the code that could point me in the right direction? At least some guidance to the main routines and the steps to modify the code.

Thanks!

**Comment by diehlpk** [comment, marked as answer]
@masoudbehzadinasab might have experiences with that. Or maybe @johntfoster
